docs/products/access/api-reference/custom-registration/custom-registration-v2.md

```diff
@@ -0,0 +1,166 @@
+# Custom Registration v2
+
+## Authentication
+
+This endpoint requires the client to provide the Client Authentication in the form of a PrivateKeyJWT
+(`urn:ietf:params:oauth:client-assertion-type:jwt-bearer`) assertion, for more info
+read [authentication methods](../../topics/authentication-methods/authentication-methods.md).
```

Comment:
Do we need to describe something about the signing method here? As you said it was very specific.

Reply:
Either here or in the place where you configure the signing keys. Are we planning to support more signing algorithms anytime soon?

Reply:
I have described its conditions here

Reply:
Nom I was not planning to